src/main/webapp/script.js

```diff
@@ -136,13 +190,40 @@ function schedule() {
   formContent.append("startDate", startDate.trim());
   formContent.append("endDate", endDate.trim());
   appendDurations(formContent);
+  formData = appendWorkingHours(formContent); // JavaScript seems pass by reference, maybe = is unnecessary, but could increase readability?
```

Comment:
I would remove the assignment as it is confusing. Your function name already implies that the passed data will be modified.
Also this is still passing by value just that the value passed is an object reference. https://stackoverflow.com/questions/518000/is-javascript-a-pass-by-reference-or-pass-by-value-language